app/server/locale/ussd.en.yml

```diff
@@ -26,15 +26,7 @@ en:
       0. Back
     send_token_reason: >
       CON Select Transfer reason
-      1. Food
-      2. Water
-      3. Energy
-      4. Education
-      5. Health
-      6. General shop
-      7. Environment
-      8. Transport
-      9. Labour
+      %options%
       10. Other
```

Comment:
I removed the categories from the tranlations and instead added the dynamic %options% variable. However send_token_reason and directory_listing differed in that send_token_reason has: "10. Other". 

Would a user select this option to other available tranfer reasons or is it used to set a tranfer as the 'other category?

Reply:
If other meas like show 'other' option it should remain if it's a separate category it should be removed

Reply:
This also affects the other option in the seed script

Reply:
I think Other is just another option in its own right, rather than a deeper breakdown